packages/pages/src/upgrade/clean_configs.ts

```diff
@@ -0,0 +1,29 @@
+import { ProjectStructure } from "../common/src/project/structure.js";
+import fs from "fs";
+import path from "path";
+
+/**
+ * cleans up all config files under sites-config except for redirects.csv
```

Comment:
We should copy redirects.csv to the root (or scoped folder) and then we can just delete the whole [scoped] folder recursively.